.markdownlint.yaml

```diff
@@ -12,3 +12,6 @@ MD024:
   allow_different_nesting: true
   # Only check sibling headings
   siblings_only: true
+
+# Allow blockquotes to be followed one by one (used as sub-sections/remarks)
+MD028: false
```

Comment:
Why? I do not see anything like this: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md028---blank-line-inside-blockquote

Reply:
If you want I can remove it

Reply:
please do it :)